packages/core/src/filters/FilterSystem.js

```diff
@@ -414,6 +418,26 @@ export default class FilterSystem extends System
         );
     }
 
+    /**
+     * Multiply input normalized coordinates to this matrix to get screen normalized coordinates.
+     *
+     * Please use `((vTextureCoord * inputSize.xy) + outputFrame.xy) / outputFrame.zw` instead.
+     *
+     * @deprecated since 5.0.0
+     * @param {PIXI.Matrix} outputMatrix - The matrix to output to.
+     * @return {PIXI.Matrix} The mapped matrix.
+     */
+    calculateNormalizedScreenSpaceMatrix(outputMatrix)
```

Comment:
Could this be added to **bundles/pixi.js/src/useDeprecated.js**? Would make sure we have a proper warning message in there.